src/backend/routers/student.ts

```diff
@@ -86,5 +86,25 @@ export const student = router({
       return result;
     }),
 
+  // TODO: get only the active IEP from the student
+  getActiveStudentIep: authenticatedProcedure
+    .input(
+      z.object({
+        student_id: z.string().uuid(),
+      })
+    )
+    .query(async (req) => {
+      const { student_id } = req.input;
+
+      const result = await req.ctx.db
+        .selectFrom("iep")
+        .where("student_id", "=", student_id)
+        .where("is_active", "=", true)
+        .selectAll()
+        .executeTakeFirst();
+
+      return result || null;
```

Comment:
type of `result` is already `Iep | null`

```suggestion
      return result;
```

Reply:
I think the reason for the return with null is that if the student has no iep data, then `result` will be `undefined`.

Reply:
oh hmm my bad, thought kysely returned null in this case

Reply:
Actually, you are right @codetheweb. I think the problem that it's not returning null is that line 104 is supposed to be `.executeTakeFirstOrThrow()`. I'm fixing this issue in the upcoming push.